source/FileSet.go

```diff
@@ -153,8 +153,9 @@ func (fs *FileSet) containsFile(file *File) bool {
 func (fs *FileSet) Contains(id string) bool {
 	if _, ok := fs.index[id]; ok {
 		return true
+	} else {
+		return false
```

Comment:
This return doesn't need to be inside an else block, just return false from the body of the function.

Reply:
@codelingobot capture return inside of unnecessary else statement

Reply:
@StagingCodeLingoBot capture return inside of unnecessary else statement

Reply:
@Codelingobotstaging capture return inside of unnecessary else statement

Reply:
@codelingobot capture capture return inside of unnecessary else statement

Reply:
@stagingcodelingobot capture return inside of unnecessary else statement